School-10

gariktepanosian opened the pull request:
I created a student section.
I modified the request links in the fragment html.
I added the fragments to the teacher's html.

gariktepanosian committed:
SCHOOL-10 I created a student section.
          I modified the request links in the fragment html.
          I added the fragments to the teacher html.

gariktepanosian committed:
SCHOOL-10 I created a student section.
          I modified the request links in the fragment html.
          I added the fragments to the teacher html.

Commit:
docker setup

schoolmanagementproject2022 committed:
Merge pull request #14 from schoolmanagementproject2022/local-branch-1

docker setup

schoolmanagementproject2022 commented on src/main/java/com/epam/edumanagementsystem/student/impl/StudentServiceImpl.java:
```diff
@@ -0,0 +1,97 @@
+    }
+
+    @Override
+    @Transactional
```
remove @Transactional from methods

schoolmanagementproject2022 commented on src/main/java/com/epam/edumanagementsystem/student/rest/api/StudentController.java:
```diff
@@ -0,0 +1,90 @@
+
+    public List<StudentDto> findAllStudents() {
+        return studentService.findAll();
+    }
```
Move this methods to the bottom of the class

gariktepanosian committed:
SCHOOL-10 I solved some issues
Bug Fixes
I changed the validation imports in Student and Teacher Entities
I changed name StudentServices to StudentService
I changed the StudentService imports

gariktepanosian committed:
Merge remote-tracking branch 'origin/SCHOOL-10' into SCHOOL-10

# Conflicts:
#	src/main/java/com/epam/edumanagementsystem/student/impl/StudentServiceImpl.java
#	src/main/java/com/epam/edumanagementsystem/student/model/entity/Student.java
#	src/main/java/com/epam/edumanagementsystem/student/rest/api/StudentController.java

gariktepanosian committed:
SCHOOL-10 I solved some issues
Bug Fixes
I changed the validation imports in Student and Teacher Entities
I changed name StudentServices to StudentService
I changed the StudentService imports

Commit:
SCHOOL-10 minor changes
